admin/models.py

```diff
@@ -1,10 +1,75 @@
-from mongoengine import Document, StringField, EmailField
+from datetime import datetime
+from mongoengine import DateTimeField, Document, EmailField, EmbeddedDocument, EmbeddedDocumentField, FloatField, \
+    IntField, ListField, ReferenceField, StringField
 
 
-class Users(Document):
-    firstname = StringField(max_length=50, required=True)
-    lastname = StringField(max_length=50, required=True)
+class Status:
+    ACTIVE = 'active'
+    DELETED = 'deleted'
```

Comment:
@Aarrtteemm123 change `DELETED = 'deleted'` to `INACTIVE = 'inactive'`, like it's up in the code